.github/workflows/coverage.yml

```diff
@@ -84,7 +84,7 @@ jobs:
 
           # CMake options & flags
           export COVERAGE_FLAGS="--coverage -O0 -fno-inline -g";
-          export CMAKE_OPTION="-DNRN_ENABLE_MPI=ON -DNRN_ENABLE_INTERVIEWS=ON -DNRN_ENABLE_PYTHON=ON -DNRN_ENABLE_PYTHON_DYNAMIC=ON -DNRN_PYTHON_DYNAMIC=${PYTHON_MIN};${PYTHON_MAX} -DNRN_ENABLE_CORENEURON=ON -DNRN_ENABLE_PROFILING=ON";
+          export CMAKE_OPTION="-DNRN_ENABLE_MPI=ON -DNRN_ENABLE_INTERVIEWS=ON -DNRN_ENABLE_PYTHON=ON -DNRN_ENABLE_PYTHON_DYNAMIC=ON -DNRN_PYTHON_DYNAMIC=${PYTHON_MIN};${PYTHON_MAX} -DNRN_ENABLE_CORENEURON=ON -DNRN_ENABLE_PROFILING=ON -DNRN_ENABLE_MODEL_TESTS=olfactory";
```

Comment:
I remember the discussion we had about the test was more elaborate. From a technical standpoint, we get `0.11%` coverage increase for an extra ~6-7 minutes of test time, so it does seem to be like using a sledgehammer for a gnat.

A couple of thoughts about this: 
* would it be possible to trim down the olfactory test even more _just_ for the context of GHA CI? if we reduce test time considerably and still achieve same coverage, that would be great.
* alternatively, could we simply rely on this test passing somewhere else so as not to put too much strain on the CI pipeline?

Reply:
Unfortunately I can't find an easy way to trim down the model so I would tend more towards the second option. This test is anyway being enabled in the CoreNEURON CI